### ISSUE TITLE
500 Internal Server Error when browsing articles from same domain

### ISSUE THREAD

**Issue by SansGuidon** [wallabag/wallabag#7262]
### Environment

* **Version**: 2.6.8
* **Installation**: Cloudron, docker based.
* **PHP version**: 8.1 ([source](https://git.cloudron.io/cloudron/wallabag-app/-/blob/v2.4.7/Dockerfile?ref_type=tags))
* **OS**: Ubuntu 22.04 (that's where Cloudron is installed)
* **Database**: PostgreSQL, not sure about the version as it's managed internally by Cloudron.
* **Parameters**:
<details>
  <summary>My <code>app/config/parameters.yml</code> is:</summary>

  ```
# This file is auto-generated during the composer install
parameters:
    database_driver: pdo_pgsql
    database_host: postgresql
    database_port: 5432
    database_name: redacted
    database_user: redacted
    database_password: redacted
    database_path: null
    database_table_prefix: wallabag_
    database_socket: null
    database_charset: utf8
    domain_name: redacted
    server_name: 'Your wallabag instance'
    mailer_dsn: 'smtp://127.0.0.1'
    locale: en
    secret: redacted
    twofactor_sender: redacted
    fosuser_registration: false
    fosuser_confirmation: false
    fos_oauth_server_access_token_lifetime: 1314000
    fos_oauth_server_refresh_token_lifetime: 1209600
    from_email: redacted
    rss_limit: 50
    rabbitmq_host: localhost
    rabbitmq_port: 5672
    rabbitmq_user: redacted
    rabbitmq_password: redacted
    rabbitmq_prefetch_count: 10
    redis_scheme: tcp
    redis_host: redacted
    redis_port: 6379
    redis_path: null
    redis_password: redacted
    sentry_dsn: null
  ```
</details>

### What steps will reproduce the bug?

The issue occurs when browsing articles from same domain.
Steps are : 
1. I click the "Show articles from the same domain.
2. If the results show more than 1 page, the error arises when trying to reach any page other than the current. So basically I can't explore articles from the same domain.
![image](https://github.com/wallabag/wallabag/assets/3721240/9ee043bf-c581-496e-aacf-a6274b82d763)
3. I'm redirected to an URL like https://wallabag.mydomain.com/domain/3494/2?redirect=/unread/list/3 and the following error is displayed 
![image](https://github.com/wallabag/wallabag/assets/3721240/fbdcdec6-a39b-4f28-8c06-0561052556b1)

**Comment by yguedidi** [comment]
@j0k3r not sure, but will investigate

**Comment by yguedidi** [comment]
@j0k3r for info, the bug was there since the feature exists.. 😅 see https://github.com/wallabag/wallabag/commit/890c7d0bfae3698e595dffc27c7027766d041776#diff-96dc7d051f1029d45165a6ac98794728d720ce28e59d6b2a0ba7673c3ef29d7dR525
here the simple fix #7266 
as it's present in 2.6, I created a 2.6.9 milestone

**Comment by SansGuidon** [comment]
Thanks a lot folks for the finding and resolution ! :-) Have a great day
